### ISSUE TITLE
Merging "Task Force Noctem" and "Task Force Noctem Vehicles" Compositions

### ISSUE THREAD

**Issue by JamesTheClarke** [CntoDev/compositions#173]
Related to issue #131.

**Problem**

Publishing composition updates on Steam Workshop is cumbersome and time-consuming. There is also a hard cap (1GB) of how much data a single account is allowed to publish under compositions - keep in mind that new versions of the same composition accumulate together towards that hard cap over time.

**Solution**

By combining the two compositions together into one it speeds up the manual efforts to update our compositions on Steam and has the side benefit of reminding people that the camo change modules for TFN also apply to the vics.

Mission makers can as always with compositions manually delete the assets they don't want to use for the mission.

**Comment by Seb105** [comment]
Personally I'm against this, our standard factions already have a lot of stuff IMO they shouldn't (Scopebox, Telestation, VISCOM). They don't neeed more.

**Comment by JamesTheClarke** [comment]
Yeah, I was on the fence about this suggestion from the start, because the vics are not necessarily going to be used every time and thus would have to be deleted in many TFN op. I'm currently trying to streamline our compositions as much as possible since updating them on Steam Workshop is kind of a pain in the ass. However, after considering it a bit more I gotta agree that this merger would probably not be all that useful.

However, functionalities such as Scopebox, Telestation and Viscom were added because they should be part of each mission, unless there's a specific reason such as time-period featuring no scopes or PvP not needing Telestation or Viscom. In the past, these items were often forgotten, even during the review process and ever since I added them plus the base clean-up module into the main faction composition those errors have been basically eliminated.

Labelling this issue as "won't fix" and closing it.
